docs/quick_start.rst

```diff
@@ -42,11 +42,18 @@ PyMeasure can also be installed with :code:`pip`.
  
 Depending on your operating system, using this method may require additional work to install the required dependencies, which include the Qt libaries.
 
-    
+Installing PyVISA
+-----------------
+Typically, communication with your instrument will happen using PyVISA, which is installed automatically.
+However, you will need a `VISA backend <https://pyvisa.readthedocs.io/en/latest/introduction/configuring.html>`__ installed to handle device communication.
+You may already know what this means and have an appropriate backend (e.g. from NI, Keysight, R&S,...) available.
```

Comment:
I find this part a little confusing, you don't know which backend to install (pyvisa-py or NI, etc). What about simply redirecting users to pyvisa documentation and remove lines 49-50 ?

Reply:
As a user, I'd prefer to not look up another documentation, but we don't want to repeat their documentation.
Therefore, I propose to keep it simple saying:
```suggestion
For example you can use the python package :code:`pyvisa-py`.
```

Reply:
I think we should give enough context, and for that we should not ignore the proprietary backends. 

I'll try to improve this. I thought it clear, simple and complete enough already. Experienced users will know the deal when they hear VISA backend, and others get simple instructions without having to read elsewhere (but the link is there if desired).

Reply:
It's good to hint at other backends and proposing (as an example) pyvisa-pay.

It's good with me

Reply:
@msmttchr good for you, too, now?

Reply:
Yes, absolutely.